readthedocs/projects/models.py

```diff
@@ -529,8 +529,18 @@ def conf_file(self, version=LATEST):
         if len(files) == 1:
             return files[0]
         for filename in files:
+            # When multiples conf.py files, we look up the first one that
+            # contains the `doc` word in its path and return this one
             if filename.find('doc', 70) != -1:
                 return filename
+
+        # If the project has more than one conf.py file but none of them have
+        # the `doc` word in the path, we raise an error informing this to the user
+        if len(files) > 1:
+            raise ProjectConfigurationError(
+                ProjectConfigurationError.MULTIPLE_CONF_FILES
+            )
+
```

Comment:
I feel like we keep changing build logic that breaks people's build. Is there a reason we are wanting to **fail** builds on this, instead of warning users? I think we probably want to warn them in this case, because changing this logic could easily break hundreds of project's builds for no real reason.

We did this with the `setup.py install` failures, and I don't think it's a good pattern to follow.

Reply:
I didn't change the logic here. I just added a better message.

Before this PR, it said "Conf.py not found" but there were more than one `conf.py` file. Now, it will fail as it was failing before but saying "Hey, too many conf.py files. We don't know which one to pick"

Reply:
hmm, interesting.